gatsby-node.js

```diff
@@ -1,17 +1,15 @@
-// const webpack = require(`webpack`)
+exports.resolvableExtensions = function () {
+  return [`.md`];
```

Comment:
Should this be `return ['.md', '.mdx']`? The Webpack test allows an `mdx` extension.